### ISSUE TITLE
Lintje doesn't detect skip ci or ci skip tag for MessageSkipBuildTag

### ISSUE THREAD

**Issue by tombruijn** [lintje/lintje#52]
## What happened?

_Describe what Lintje did._

How I called the Lintje command:
```sh
$ lintje
```

The output I received:
```
Hint[MessageSkipBuildTag]: Consider skipping the build for a text change that does not impact the test suite
  b765690: Fix syntax error in config object in README
    |
    | README.md
    | ^^^^^^^^^ Only text files were changed
   ~~~
  9 | [skip ci]
    | +++++++++ Add the skip build tag to the commit message
    |
    = help: https://r.lintje.dev/r/MessageSkipBuildTag
```

## What was supposed to happen?
[This commit](https://github.com/appsignal/appsignal-nodejs/commit/b765690379d89b32f76b6169aacfafc13034119b) has a `[ci skip]` tag, so it shouldn't have prompted this hint.

### Version

lintje 0.11.1
aarch64-apple-darwin

### What Operating System are you seeing the problem on?

macOS

### How are you running Lintje?

Directly on host machine (Default)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct
